modules/tutorials/pages/install-couchbase-server.adoc

```diff
@@ -10,13 +10,14 @@ include::partial$tutorial-globals.adoc[]
 {description}
 
 == Installation
-Couchbase can be installed as either:
 
-* A standalone application
-* A Docker instance
+The easiest way to get started with Couchbase is to use the https://www.couchbase.com/products/capella[Capella], which allows you to set up, administer and run Couchbase clusters as a cloud service.
```

Comment:
The easiest way to get started with Couchbase is to use  https://www.couchbase.com/products/capella[Capella],

^ omit second definite article.

Also needs serial comma (after administer).